src/fdk_rdf_parser/organizations/utils.py

```diff
@@ -13,3 +14,9 @@ def organizationUrl(orgnr: Optional[str]) -> str:
         if orgnr is not None
         else f"{baseOrgUrl}/organizations"
     )
+
+
+def organisationNumberFromUri(uri: str) -> Optional[str]:
+    enhetsregisteretUriBase = "https://data.brreg.no/enhetsregisteret/api/enheter"
+    match = re.compile(f"^{enhetsregisteretUriBase}/(\\d{9})$").match(uri)
+    return match.group(1) if match is not None else None
```

Comment:
use `search()` for positive look-behind, e.g so we can `group(0)` instead of `group(last)`

```suggestion
    match = re.compile(f"^{enhetsregisteretUriBase}/(\\d{9})$").search(uri)
    return match.group(0) if match else None
```

Reply:
Done